config/pipeline.yaml

```diff
@@ -447,6 +447,20 @@ jobs:
         patchlevel: 7
     kcidb_test_suite: kselftest.dt
 
+  kselftest-discoverable-devices-probe:
+    template: kselftest.jinja2
+    kind: test
+    params:
+      nfsroot: 'http://storage.kernelci.org/images/rootfs/debian/bookworm-kselftest/20240221.0/{debarch}' # TODO: update once new rootfs with platform-test-parameters is generated
```

Comment:
Try https://storage.staging.kernelci.org/rootfs/bookworm-kselftest/20240704.0 (just to spin this up on staging)

Reply:
Done. Leaving the comment so I remember to update it to a production URL before merging.